lib/rexml/document.rb

```diff
@@ -447,7 +447,21 @@ def document
       self
     end
 
+    attr_accessor :namespaces_cache
```

Comment:
Can we hide this?

(Hmm. Is adding a cache related API to document a bad idea...?)

Reply:
OK
I see.